src/datagen/generated/minecolonies/data/minecolonies/tags/blocks/pathblocks.json

```diff
@@ -28,6 +28,9 @@
     "minecraft:chiseled_nether_bricks",
     "minecraft:gravel",
     "minecraft:dirt_path",
+    "minecraft:mud_bricks",
```

Comment:
Can you already add the missing strong brick, stone brick stairs, cracked etc?

Reply:
Cracked stone bricks are already in there via the STONE_BRICKS tag.

Reply:
I've added a [path tag audit](https://docs.google.com/spreadsheets/d/1i6EQuPuxVWDmH2Z9_CHiXOmDG5sXVF2cscHg6lwXs9o/edit?gid=1294272618#gid=1294272618) now, which makes it easier to see which blocks are paths and which aren't.